### ISSUE TITLE
[0.2] Test error with casa_low_level_io

### ISSUE THREAD

**Issue by olebole** [radio-astro-tools/casa-formats-io#53]
When trying to package version 0.2 for Debian, I observed the following error:
```
============================= test session starts ==============================
platform linux -- Python 3.10.7, pytest-7.1.2, pluggy-1.0.0+repack
Matplotlib: 3.5.2
Freetype: 2.12.1
rootdir: /home/ole/Projects/2011/debian/python/astropy/casa-formats-io, configfile: setup.cfg
plugins: doctestplus-0.12.0, mock-3.8.2, asdf-2.12.0, filter-subpackage-0.1.1, hypothesis-6.36.0, mpl-0.11, openfiles-0.5.0, cov-3.0.0, astropy-header-0.2.2, astropy-0.10.0, arraydiff-0.5.0, remotedata-0.3.3
collected 77 items / 1 error

==================================== ERRORS ====================================
_ ERROR collecting .pybuild/cpython3_3.10_casa-formats-io/build/casa_formats_io/casa_low_level_io.py _
/usr/lib/python3/dist-packages/pytest_doctestplus/plugin.py:246: in collect
    module = import_path(fspath, root=self.config.rootpath)
/usr/lib/python3/dist-packages/_pytest/pathlib.py:556: in import_path
    raise ImportPathMismatchError(module_name, module_file, path)
E   _pytest.pathlib.ImportPathMismatchError: ('casa_formats_io.casa_low_level_io', '…/casa-formats-io/.pybuild/cpython3_3.10_casa-formats-io/build/casa_formats_io/casa_low_level_io', PosixPath('/…/casa-formats-io/.pybuild/cpython3_3.10_casa-formats-io/build/casa_formats_io/casa_low_level_io.py'))
```
What I am wondering: there is both a file `casa_low_level_io.py` and a directory `casa_low_level_io/` in `casa-formats-io/`; maybe this causes trouble?

**Comment by astrofrog** [comment]
You are right, it looks like I forgot to remove this in https://github.com/radio-astro-tools/casa-formats-io/pull/24 - I should remove that and release a bugfix version.

**Comment by astrofrog** [comment]
@olebole - is there a way to easily check that https://github.com/radio-astro-tools/casa-formats-io/pull/54 would work for you?

**Comment by olebole** [comment]
I checked this by removing  `casa_formats_io/casa_low_level_io.py`, and then the package builds fine.

**Comment by astrofrog** [comment]
Thanks! I've triggered a build of v0.2.1 so if all goes well it should be published in an hour or so.

**Comment by astrofrog** [comment]
Closing this as should be fixed in v0.2.1
